spec/features/user_updates_cart_spec.rb

```diff
@@ -27,13 +27,14 @@
     visit item_path(@item2)
     click_on "Add to cart"
 
-    fill_in "Quantity:", :match => :first, with: 4
+    items = page.all("li#cart-item")
+    items[0].fill_in('Quantity:', with: 2)
+    items[1].fill_in('Quantity:', with: 4)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.